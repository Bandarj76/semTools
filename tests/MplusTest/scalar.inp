TITLE: multigroup scalar invariance model
DATA:
  FILE = "imps.dat";
  TYPE IS IMPUTATION;
VARIABLE:
  NAMES = x1 x2 x3 x4 x5 x6 x7 x8 x9 school;
  USEVAR = x1-x9;
  GROUPING = school (1 = Grant_White   2 = Pasteur);
MODEL:
  f1 BY x1-x3;
  f2 BY x4-x6;
  f3 BY x7-x9;
MODEL Grant_White:
  [f1 f2 f3];
MODEL Pasteur:
  [f1@0 f2@0 f3@0];
