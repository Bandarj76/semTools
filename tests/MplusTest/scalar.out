Mplus VERSION 7.11
MUTHEN & MUTHEN
01/29/2017   7:19 PM

INPUT INSTRUCTIONS

  TITLE: multigroup scalar invariance model
  DATA:
    FILE = "imps.dat";
    TYPE IS IMPUTATION;
  VARIABLE:
    NAMES = x1 x2 x3 x4 x5 x6 x7 x8 x9 school;
    USEVAR = x1-x9;
    GROUPING = school (1 = Grant_White   2 = Pasteur);
  MODEL:
    f1 BY x1-x3;
    f2 BY x4-x6;
    f3 BY x7-x9;
  MODEL Grant_White:
    [f1 f2 f3];
  MODEL Pasteur:
    [f1@0 f2@0 f3@0];



INPUT READING TERMINATED NORMALLY



multigroup scalar invariance model

SUMMARY OF ANALYSIS

Number of groups                                                 2
Average number of observations
   Group GRANT_WHITE                                           145
   Group PASTEUR                                               156
   Total sample size                                           301

Number of replications
    Requested                                                   20
    Completed                                                   20

Number of dependent variables                                    9
Number of independent variables                                  0
Number of continuous latent variables                            3

Observed dependent variables

  Continuous
   X1          X2          X3          X4          X5          X6
   X7          X8          X9

Continuous latent variables
   F1          F2          F3

Variables with special functions

  Grouping variable     SCHOOL

Estimator                                                       ML
Information matrix                                        OBSERVED
Maximum number of iterations                                  1000
Convergence criterion                                    0.500D-04
Maximum number of steepest descent iterations                   20

Input data file(s)
  Multiple data files from
    imps.dat

Input data format  FREE


SAMPLE STATISTICS

NOTE:  These are average results over 20 data sets.


     SAMPLE STATISTICS FOR GRANT_WHITE


           Means
              X1            X2            X3            X4            X5
              ________      ________      ________      ________      ________
      1         4.930         6.200         1.996         3.317         4.800


           Means
              X6            X7            X8            X9
              ________      ________      ________      ________
      1         2.469         3.921         5.488         5.406


           Covariances
              X1            X2            X3            X4            X5
              ________      ________      ________      ________      ________
 X1             1.319
 X2             0.414         1.226
 X3             0.534         0.478         1.073
 X4             0.440         0.283         0.381         1.257
 X5             0.278         0.193         0.332         0.931         1.328
 X6             0.412         0.244         0.407         0.906         0.900
 X7             0.123         0.076         0.080         0.241         0.209
 X8             0.370         0.195         0.259         0.122         0.121
 X9             0.473         0.266         0.400         0.364         0.263


           Covariances
              X6            X7            X8            X9
              ________      ________      ________      ________
 X6             1.280
 X7             0.208         1.062
 X8             0.143         0.633         1.094
 X9             0.365         0.359         0.499         0.893


           Correlations
              X1            X2            X3            X4            X5
              ________      ________      ________      ________      ________
 X1             1.000
 X2             0.326         1.000
 X3             0.449         0.417         1.000
 X4             0.342         0.228         0.328         1.000
 X5             0.210         0.151         0.278         0.720         1.000
 X6             0.317         0.195         0.347         0.714         0.690
 X7             0.104         0.066         0.075         0.209         0.176
 X8             0.308         0.168         0.239         0.104         0.100
 X9             0.436         0.254         0.409         0.344         0.242


           Correlations
              X6            X7            X8            X9
              ________      ________      ________      ________
 X6             1.000
 X7             0.179         1.000
 X8             0.121         0.587         1.000
 X9             0.341         0.368         0.505         1.000


     SAMPLE STATISTICS FOR PASTEUR


           Means
              X1            X2            X3            X4            X5
              ________      ________      ________      ________      ________
      1         4.941         5.984         2.487         2.823         4.055


           Means
              X6            X7            X8            X9
              ________      ________      ________      ________
      1         1.922         4.432         5.563         5.421


           Covariances
              X1            X2            X3            X4            X5
              ________      ________      ________      ________      ________
 X1             1.395
 X2             0.402         1.504
 X3             0.620         0.477         1.346
 X4             0.568         0.089         0.165         1.320
 X5             0.413         0.134         0.098         0.946         1.460
 X6             0.498         0.194         0.222         0.755         0.837
 X7             0.046        -0.204        -0.025         0.322         0.225
 X8             0.165         0.039         0.152         0.147         0.087
 X9             0.370         0.181         0.265         0.255         0.231


           Covariances
              X6            X7            X8            X9
              ________      ________      ________      ________
 X6             0.974
 X7             0.219         1.170
 X8             0.206         0.426         0.952
 X9             0.241         0.344         0.367         0.956


           Correlations
              X1            X2            X3            X4            X5
              ________      ________      ________      ________      ________
 X1             1.000
 X2             0.278         1.000
 X3             0.453         0.335         1.000
 X4             0.419         0.063         0.124         1.000
 X5             0.290         0.091         0.070         0.682         1.000
 X6             0.427         0.160         0.194         0.666         0.702
 X7             0.036        -0.154        -0.020         0.259         0.172
 X8             0.143         0.032         0.134         0.131         0.074
 X9             0.321         0.151         0.234         0.227         0.195


           Correlations
              X6            X7            X8            X9
              ________      ________      ________      ________
 X6             1.000
 X7             0.205         1.000
 X8             0.214         0.404         1.000
 X9             0.249         0.325         0.385         1.000


MODEL FIT INFORMATION

Number of Free Parameters                       48

Loglikelihood

          H0 Value                       -3688.590
          H1 Value                       -3604.529

*   The loglikelihood cannot be used directly for chi-square testing with
    imputed data.

Information Criteria

          Akaike (AIC)                    7473.179
          Bayesian (BIC)                  7651.121
          Sample-Size Adjusted BIC        7498.892
            (n* = (n + 2) / 24)

Chi-Square Test of Model Fit

          Value                            125.508
          Degrees of Freedom                    60
          P-Value                           0.0000

RMSEA (Root Mean Square Error Of Approximation)

          Estimate                           0.085
          90 Percent C.I.                    0.064  0.106
          Probability RMSEA <= .05           0.004

CFI/TLI

          CFI                                0.900
          TLI                                0.880

Chi-Square Test of Model Fit for the Baseline Model

          Value                            728.288
          Degrees of Freedom                    72
          P-Value                           0.0000

SRMR (Standardized Root Mean Square Residual)

          Value                              0.087



MODEL RESULTS

                                                    Two-Tailed   Rate of
                    Estimate       S.E.  Est./S.E.    P-Value    Missing

Group GRANT_WHITE

 F1       BY
    X1                 1.000      0.000    999.000    999.000      0.000
    X2                 0.593      0.115      5.152      0.000      0.066
    X3                 0.832      0.141      5.905      0.000      0.080

 F2       BY
    X4                 1.000      0.000    999.000    999.000      0.000
    X5                 1.064      0.083     12.874      0.000      0.291
    X6                 0.953      0.063     15.097      0.000      0.114

 F3       BY
    X7                 1.000      0.000    999.000    999.000      0.000
    X8                 1.143      0.161      7.095      0.000      0.129
    X9                 0.998      0.330      3.022      0.003      0.390

 F2       WITH
    F1                 0.406      0.098      4.136      0.000      0.023

 F3       WITH
    F1                 0.313      0.084      3.728      0.000      0.081
    F2                 0.206      0.078      2.641      0.008      0.097

 Means
    F1                -0.156      0.125     -1.244      0.213      0.006
    F2                 0.589      0.118      5.012      0.000      0.017
    F3                -0.150      0.103     -1.454      0.146      0.138

 Intercepts
    X1                 5.005      0.091     54.844      0.000      0.003
    X2                 6.156      0.080     76.936      0.000      0.006
    X3                 2.276      0.090     25.264      0.000      0.007
    X4                 2.768      0.086     32.209      0.000      0.009
    X5                 4.117      0.115     35.848      0.000      0.276
    X6                 1.917      0.078     24.514      0.000      0.005
    X7                 4.236      0.080     52.933      0.000      0.051
    X8                 5.613      0.078     71.712      0.000      0.083
    X9                 5.491      0.091     60.054      0.000      0.306

 Variances
    F1                 0.683      0.161      4.244      0.000      0.041
    F2                 0.901      0.139      6.495      0.000      0.040
    F3                 0.465      0.144      3.230      0.001      0.237

 Residual Variances
    X1                 0.688      0.136      5.070      0.000      0.064
    X2                 0.957      0.128      7.502      0.000      0.013
    X3                 0.614      0.115      5.348      0.000      0.042
    X4                 0.323      0.067      4.795      0.000      0.137
    X5                 0.409      0.086      4.744      0.000      0.227
    X6                 0.408      0.071      5.737      0.000      0.115
    X7                 0.675      0.141      4.773      0.000      0.250
    X8                 0.455      0.147      3.100      0.002      0.283
    X9                 0.456      0.141      3.234      0.001      0.348

Group PASTEUR

 F1       BY
    X1                 1.000      0.000    999.000    999.000      0.000
    X2                 0.593      0.115      5.152      0.000      0.066
    X3                 0.832      0.141      5.905      0.000      0.080

 F2       BY
    X4                 1.000      0.000    999.000    999.000      0.000
    X5                 1.064      0.083     12.874      0.000      0.291
    X6                 0.953      0.063     15.097      0.000      0.114

 F3       BY
    X7                 1.000      0.000    999.000    999.000      0.000
    X8                 1.143      0.161      7.095      0.000      0.129
    X9                 0.998      0.330      3.022      0.003      0.390

 F2       WITH
    F1                 0.396      0.108      3.678      0.000      0.081

 F3       WITH
    F1                 0.172      0.074      2.329      0.020      0.139
    F2                 0.196      0.067      2.949      0.003      0.075

 Means
    F1                 0.000      0.000    999.000    999.000      0.000
    F2                 0.000      0.000    999.000    999.000      0.000
    F3                 0.000      0.000    999.000    999.000      0.000

 Intercepts
    X1                 5.005      0.091     54.844      0.000      0.003
    X2                 6.156      0.080     76.936      0.000      0.006
    X3                 2.276      0.090     25.264      0.000      0.007
    X4                 2.768      0.086     32.209      0.000      0.009
    X5                 4.117      0.115     35.848      0.000      0.276
    X6                 1.917      0.078     24.514      0.000      0.005
    X7                 4.236      0.080     52.933      0.000      0.051
    X8                 5.613      0.078     71.712      0.000      0.083
    X9                 5.491      0.091     60.054      0.000      0.306

 Variances
    F1                 0.767      0.192      3.985      0.000      0.055
    F2                 0.825      0.129      6.415      0.000      0.039
    F3                 0.341      0.102      3.329      0.001      0.159

 Residual Variances
    X1                 0.577      0.160      3.618      0.000      0.068
    X2                 1.292      0.161      8.025      0.000      0.006
    X3                 0.931      0.150      6.188      0.000      0.027
    X4                 0.481      0.080      6.048      0.000      0.098
    X5                 0.496      0.109      4.545      0.000      0.318
    X6                 0.252      0.057      4.390      0.000      0.152
    X7                 0.865      0.135      6.422      0.000      0.107
    X8                 0.537      0.116      4.620      0.000      0.200
    X9                 0.598      0.129      4.626      0.000      0.263


QUALITY OF NUMERICAL RESULTS

     Average Condition Number for the Information Matrix      0.233E-02
       (ratio of smallest to largest eigenvalue)


TECHNICAL 1 OUTPUT


     PARAMETER SPECIFICATION FOR GRANT_WHITE


           NU
              X1            X2            X3            X4            X5
              ________      ________      ________      ________      ________
      1           1             2             3             4             5


           NU
              X6            X7            X8            X9
              ________      ________      ________      ________
      1           6             7             8             9


           LAMBDA
              F1            F2            F3
              ________      ________      ________
 X1                 0             0             0
 X2                10             0             0
 X3                11             0             0
 X4                 0             0             0
 X5                 0            12             0
 X6                 0            13             0
 X7                 0             0             0
 X8                 0             0            14
 X9                 0             0            15


           THETA
              X1            X2            X3            X4            X5
              ________      ________      ________      ________      ________
 X1                16
 X2                 0            17
 X3                 0             0            18
 X4                 0             0             0            19
 X5                 0             0             0             0            20
 X6                 0             0             0             0             0
 X7                 0             0             0             0             0
 X8                 0             0             0             0             0
 X9                 0             0             0             0             0


           THETA
              X6            X7            X8            X9
              ________      ________      ________      ________
 X6                21
 X7                 0            22
 X8                 0             0            23
 X9                 0             0             0            24


           ALPHA
              F1            F2            F3
              ________      ________      ________
      1          25            26            27


           BETA
              F1            F2            F3
              ________      ________      ________
 F1                 0             0             0
 F2                 0             0             0
 F3                 0             0             0


           PSI
              F1            F2            F3
              ________      ________      ________
 F1                28
 F2                29            30
 F3                31            32            33


     PARAMETER SPECIFICATION FOR PASTEUR


           NU
              X1            X2            X3            X4            X5
              ________      ________      ________      ________      ________
      1           1             2             3             4             5


           NU
              X6            X7            X8            X9
              ________      ________      ________      ________
      1           6             7             8             9


           LAMBDA
              F1            F2            F3
              ________      ________      ________
 X1                 0             0             0
 X2                10             0             0
 X3                11             0             0
 X4                 0             0             0
 X5                 0            12             0
 X6                 0            13             0
 X7                 0             0             0
 X8                 0             0            14
 X9                 0             0            15


           THETA
              X1            X2            X3            X4            X5
              ________      ________      ________      ________      ________
 X1                34
 X2                 0            35
 X3                 0             0            36
 X4                 0             0             0            37
 X5                 0             0             0             0            38
 X6                 0             0             0             0             0
 X7                 0             0             0             0             0
 X8                 0             0             0             0             0
 X9                 0             0             0             0             0


           THETA
              X6            X7            X8            X9
              ________      ________      ________      ________
 X6                39
 X7                 0            40
 X8                 0             0            41
 X9                 0             0             0            42


           ALPHA
              F1            F2            F3
              ________      ________      ________
      1           0             0             0


           BETA
              F1            F2            F3
              ________      ________      ________
 F1                 0             0             0
 F2                 0             0             0
 F3                 0             0             0


           PSI
              F1            F2            F3
              ________      ________      ________
 F1                43
 F2                44            45
 F3                46            47            48


     STARTING VALUES FOR GRANT_WHITE


           NU
              X1            X2            X3            X4            X5
              ________      ________      ________      ________      ________
      1         4.930         6.200         1.996         3.317         4.717


           NU
              X6            X7            X8            X9
              ________      ________      ________      ________
      1         2.469         3.921         5.488         5.428


           LAMBDA
              F1            F2            F3
              ________      ________      ________
 X1             1.000         0.000         0.000
 X2             0.896         0.000         0.000
 X3             1.155         0.000         0.000
 X4             0.000         1.000         0.000
 X5             0.000         0.991         0.000
 X6             0.000         0.937         0.000
 X7             0.000         0.000         1.000
 X8             0.000         0.000         1.473
 X9             0.000         0.000         0.800


           THETA
              X1            X2            X3            X4            X5
              ________      ________      ________      ________      ________
 X1             0.500
 X2             0.000         0.500
 X3             0.000         0.000         0.500
 X4             0.000         0.000         0.000         0.500
 X5             0.000         0.000         0.000         0.000         0.500
 X6             0.000         0.000         0.000         0.000         0.000
 X7             0.000         0.000         0.000         0.000         0.000
 X8             0.000         0.000         0.000         0.000         0.000
 X9             0.000         0.000         0.000         0.000         0.000


           THETA
              X6            X7            X8            X9
              ________      ________      ________      ________
 X6             0.500
 X7             0.000         0.500
 X8             0.000         0.000         0.500
 X9             0.000         0.000         0.000         0.500


           ALPHA
              F1            F2            F3
              ________      ________      ________
      1         0.000         0.000         0.000


           BETA
              F1            F2            F3
              ________      ________      ________
 F1             0.000         0.000         0.000
 F2             0.000         0.000         0.000
 F3             0.000         0.000         0.000


           PSI
              F1            F2            F3
              ________      ________      ________
 F1             0.050
 F2             0.000         0.050
 F3             0.000         0.000         0.050


     STARTING VALUES FOR PASTEUR


           NU
              X1            X2            X3            X4            X5
              ________      ________      ________      ________      ________
      1         4.930         6.200         1.996         3.317         4.717


           NU
              X6            X7            X8            X9
              ________      ________      ________      ________
      1         2.469         3.921         5.488         5.428


           LAMBDA
              F1            F2            F3
              ________      ________      ________
 X1             1.000         0.000         0.000
 X2             0.769         0.000         0.000
 X3             1.186         0.000         0.000
 X4             0.000         1.000         0.000
 X5             0.000         1.133         0.000
 X6             0.000         0.863         0.000
 X7             0.000         0.000         1.000
 X8             0.000         0.000         1.357
 X9             0.000         0.000         0.892


           THETA
              X1            X2            X3            X4            X5
              ________      ________      ________      ________      ________
 X1             0.500
 X2             0.000         0.500
 X3             0.000         0.000         0.500
 X4             0.000         0.000         0.000         0.500
 X5             0.000         0.000         0.000         0.000         0.500
 X6             0.000         0.000         0.000         0.000         0.000
 X7             0.000         0.000         0.000         0.000         0.000
 X8             0.000         0.000         0.000         0.000         0.000
 X9             0.000         0.000         0.000         0.000         0.000


           THETA
              X6            X7            X8            X9
              ________      ________      ________      ________
 X6             0.500
 X7             0.000         0.500
 X8             0.000         0.000         0.500
 X9             0.000         0.000         0.000         0.500


           ALPHA
              F1            F2            F3
              ________      ________      ________
      1         0.000         0.000         0.000


           BETA
              F1            F2            F3
              ________      ________      ________
 F1             0.000         0.000         0.000
 F2             0.000         0.000         0.000
 F3             0.000         0.000         0.000


           PSI
              F1            F2            F3
              ________      ________      ________
 F1             0.050
 F2             0.000         0.050
 F3             0.000         0.000         0.050


     Beginning Time:  19:19:35
        Ending Time:  19:19:36
       Elapsed Time:  00:00:01



MUTHEN & MUTHEN
3463 Stoner Ave.
Los Angeles, CA  90066

Tel: (310) 391-9971
Fax: (310) 391-8971
Web: www.StatModel.com
Support: Support@StatModel.com

Copyright (c) 1998-2013 Muthen & Muthen
